tests/pkcli/ci1_test.py

```diff
@@ -14,14 +14,16 @@ def test_check_prints_dot_prefix():
     from pykern.pkcli import ci, test
     from pykern import pkunit
 
-    ci._EXCLUDE_FILES = re.compile(
-        re.sub(
-            f"/{test.SUITE_D}/.*(?:{pkunit.DATA_DIR_SUFFIX}|{pkunit.WORK_DIR_SUFFIX})/",
-            f"/{test.SUITE_D}/.*{pkunit.DATA_DIR_SUFFIX}/",
-            ci._EXCLUDE_FILES.pattern
-        )
-    )
-
+    _unexclude_work_dir()
```

Comment:
I don't think this should be necessary with relative paths, because pwd will be inside _work.